ui/app/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js

```diff
@@ -41,7 +41,7 @@ export default class ImportWithSeedPhrase extends PureComponent {
       return ''
     }
 
-    const words = trimmed.match(/\w+/g)
+    const words = trimmed.toLowerCase().match(/\w+/g)
```

Comment:
Could we instead use the case-insensitive flag?

Reply:
We could -- I wasn't sure if downstream usages of `words` (i.e. lookup into bip39 word list, etc.) would behave unexpectedly if `words` were no longer lowercased. Let me know your thoughts -- if we're OK changing the return contract here.

Reply:
Ah, yes—I misunderstood what the PR was supposed to do. This is good.